### PR TITLE
Use `ebryn/backburner.js#cleanup-build`

### PR DESCRIPTION
**Pull request by seanpdoyle**
Ignore me.

### CODE REVIEW

**Comment by seanpdoyle** [comment]
@rwjblue I'm seeing this failure locally:

```
Build failed.
File: htmlbars-syntax/handlebars/compiler/base.js
htmlbars-syntax/handlebars/compiler/base.js: Unexpected token (4:7)
SyntaxError: htmlbars-syntax/handlebars/compiler/base.js: Unexpected token (4:7)
  2 | import AST from "./ast";
  3 | import WhitespaceControl from "./whitespace-control";
> 4 | module Helpers from "./helpers";
    |        ^
  5 | import { extend } from "../utils";
  6 | 
  7 | export { parser };
```

which is fun, because I can't seem to track down the file in question. Could you point me in the right direction?

**Comment by seanpdoyle** [comment]
```js
// node_modules/htmlbars/dist/cjs/htmlbars-syntax/handlebars/compiler/base.js
var Helpers = __es6_transpiler_build_module_object__("Helpers", require("./helpers"));
```

is the compiled output in question, but the source file doesn't exactly exist

**Comment by seanpdoyle** [comment]
https://github.com/wycats/handlebars.js/blob/37a664bf6438bde0ee827a68da4cbd95a21cf52e/lib/handlebars/compiler/base.js#L4

this is fun :cyclone:

**Comment by seanpdoyle** [comment]
attempted to correct in https://github.com/wycats/handlebars.js/pull/993

**Comment by seanpdoyle** [comment]
@rwjblue seems like Travis' cache isn't busted by npm or bower:

https://travis-ci.org/emberjs/ember.js/jobs/59451384

**Comment by rwjblue** [comment]
I can clear the Travis cache, but Ember master cannot use `emberjs-build@0.1.0` since it requires the latest updates to Handlebars (that only works in idempotent-rerender).

**Comment by seanpdoyle** [comment]
@rwjblue got it. should I roll `emberjs-build` back to `0.0.48` and keep going?

**Comment by stefanpenner** [comment]
@seanpdoyle let me know, if that branch (on backburner) is ready to be merged. I can pull the switch.

**Comment by seanpdoyle** [comment]
@stefanpenner which branch? (https://github.com/ebryn/backburner.js/pulls)

I think it's been merged.

**Comment by stefanpenner** [comment]
@seanpdoyle ah!

**Comment by seanpdoyle** [comment]
@stefanpenner @rwjblue could we clear the cache so that these new versions stick?

**Comment by stefanpenner** [comment]
@seanpdoyle i've blown the cache. I wonder if travis is interleaving caches per branch...

**Comment by stefanpenner** [comment]
purged again, rerunning.

**Comment by seanpdoyle** [comment]
@stefanpenner doesn't seem to have been purged: https://travis-ci.org/emberjs/ember.js/jobs/59469948#L137-L142

**Comment by stefanpenner** [comment]
@seanpdoyle i think travis is borked, i blew all of the caches away

**Comment by seanpdoyle** [comment]
@stefanpenner @rwjblue we're back to this:

https://travis-ci.org/emberjs/ember.js/jobs/59469948#L137-L142

File: package.json [hunk 1]
```diff
@@ -19,15 +19,15 @@
...
     "express": "^4.5.0",
     "github": "^0.2.3",
     "glob": "~4.3.2",
-    "htmlbars": "0.11.3",
+    "htmlbars": "~0.13.0",
     "qunit-extras": "^1.3.0",
     "qunitjs": "^1.16.0",
     "route-recognizer": "0.1.5",
-    "rsvp": "~3.0.6",
+    "rsvp": "https://github.com/tildeio/rsvp.js.git",
```
Comment (seanpdoyle) [inline package.json]: Do I need these to be master in both files?

**Comment by seanpdoyle** [comment]
screen shot 2015-04-21 at 9 25 46 pm

**Comment by seanpdoyle** [comment]
@stefanpenner could you purge the cache again?

**Comment by stefanpenner** [comment]
> @stefanpenner could you purge the cache again?

i think travis fixed their underlying issue, restarting tests.

File: package.json (hunk 1 above)
Comment (stefanpenner) [inline package.json]: i believe this one is just for build tooling

**Comment by stefanpenner** [comment]
> @stefanpenner @rwjblue somehow we're back to this:

These UMD files, are not for esperanto to consume. They exist as input files to the old es6-module-transpilers bundle format. We should just filter then out of the tree before passing to esperanto.

File: bower.json [hunk 2]
```diff
@@ -6,10 +6,10 @@
     "qunit-phantom-runner": "jonkemp/qunit-phantomjs-runner#1.2.0"
   },
   "devDependencies": {
-    "backburner": "https://github.com/ebryn/backburner.js.git#f4bd6a2df221240ed36d140f0c53c036a7ecacad",
-    "rsvp": "https://github.com/tildeio/rsvp.js.git#3.0.14",
+    "backburner": "https://github.com/ebryn/backburner.js.git",
```
Comment (seanpdoyle) [inline bower.json]: @stefanpenner this is using `master` of backburner.

Should I peg it to a particular commit?

**Comment by stefanpenner** [comment]
Ah. I'll poke around

**Pull request closed without merging**